losses/base.py

```diff
@@ -196,20 +221,13 @@ def __init__(self, loss_name, model, device, loss_fn, loader, optim, recover=Non
         self.recover = recover
         self.batch_transforms = batch_transforms
         self.callbacks = callbacks
+        self.model_loader = ModelLoader(loss_name)
```

Comment:
If the loss name is not unsed anywhere else in the class, maybe we can merge `loss_name` and `recover` into a single `model_loader` parameter.
I need to check if this is possible.